tests/unit_tests/dft/source/descriptor_tests.cpp

```diff
@@ -401,53 +395,89 @@ inline void set_readonly_values(sycl::queue& sycl_queue) {
     EXPECT_THROW(descriptor.set_value(oneapi::mkl::dft::config_param::COMMIT_STATUS,
                                       oneapi::mkl::dft::config_value::UNCOMMITTED),
                  oneapi::mkl::invalid_argument);
+}
 
+template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain>
+static void get_commited(sycl::queue& sycl_queue) {
+    oneapi::mkl::dft::descriptor<precision, domain> descriptor{ default_1d_lengths };
     commit_descriptor(descriptor, sycl_queue);
+
+    oneapi::mkl::dft::config_value commit_status;
+    descriptor.get_value(oneapi::mkl::dft::config_param::COMMIT_STATUS, &commit_status);
+    EXPECT_EQ(commit_status, oneapi::mkl::dft::config_value::COMMITTED);
 }
 
 template <oneapi::mkl::dft::precision precision, oneapi::mkl::dft::domain domain>
-int test(sycl::device* dev) {
-    sycl::queue sycl_queue(*dev, exception_handler);
+static int test_getter_setter() {
+    set_and_get_lengths<precision, domain>();
+    set_and_get_strides<precision, domain>();
+    set_and_get_values<precision, domain>();
+    get_readonly_values<precision, domain>();
+    set_readonly_values<precision, domain>();
```

Comment:
test changes should probably be in a separate PR. Did your other changes show any issues that necessitated these test changes?

Reply:
That's fair, I can split it. This came up simply because some tests had an unused queue argument.

Reply:
Done with https://github.com/oneapi-src/oneMKL/pull/290/commits/2a89e405d2741d5a3319206393aa59436ab8bb04